### PR TITLE
Update idol-library dependency version

### PR DESCRIPTION
**Pull request by christopher-rayson-mf**
Attempt to fix errors when deploying multiple components in a single chart, which I believe are down to inconsistent definitions of common idol-library templates by different versions of idol-library, pulled in as a dependency by various subcharts.

### CODE REVIEW

**Review by james-poole-mf — APPROVED** [review]
This all seems reasonable, if more drastic changes than I might have naively expected... are they all solely due to idol-library version upgrades?

**Comment by christopher-rayson-mf** [comment]
> This all seems reasonable, if more drastic changes than I might have naively expected... are they all solely due to idol-library version upgrades?

Yes, the hope is there's no actual functional change to the default deployment, motivation is just to bump the underlying idol-library to something compatible with the latest nifi chart. I've followed the pattern of #41, #42; maybe there is a less intrusive way to make the changes if you don't need any of the extra bells/whistles from the newer library chart, but naïvely bumping the idol-library dependency version caused templating errors, so I've just copied previous upgrades from pre- to post-0.7.0, which seems to be the watershed in terms of incremental updating.

**Comment by james-poole-mf** [comment]
Thanks Chris. This looks good.

File: helm/idol-qms/values.schema.json [hunk 1]
```diff
@@ -109,6 +83,13 @@
                 },
                 "host": {
                     "type": "string"
+                },
+                "proxyBodySize":{
+                    "type":"string",
+                    "pattern": "^\\d+(?:\\.\\d+)?(?:Ki|Mi|Gi|Ti|Pi|Ei|m|k|M|G|T|P|E)?$"
+                },
+                "annotations": {
+                    "type": "object"
```
Comment (joe-wood-opentext) [inline helm/idol-qms/values.schema.json:92]: Do these want sensible defaults in `values.yaml` and docs comments explaining what they are?

File: helm/idol-qms/values.yaml [hunk 2]
```diff
@@ -45,6 +50,22 @@ additionalVolumes: []
...
+# -- Optional configMap name holding extra environnment variables for container
+envConfigMap: ""
+
+# -- Optional PodSecurityContext (see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podsecuritycontext-v1-core)
+podSecurityContext:
+  # -- enable PodSecurityContext. Setting to false omits.
+  enabled: false
+  fsGroup: 0
+  runAsUser: 1000
+  runAsGroup: 0
```
Comment (joe-wood-opentext) [inline helm/idol-qms/values.yaml:62]: Do any of these want docs comments either? Doesn't look like this was done in previous upgrades - not sure if that's intentional or not?

Comment (christopher-rayson-mf) [reply helm/idol-qms/values.yaml:62]: The parent `podSecurityContext` comment points to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podsecuritycontext-v1-core which covers all the elements — is that sufficient?

Comment (joe-wood-opentext) [reply helm/idol-qms/values.yaml:62]: Ah fair, yeah if there's aren't internal variables then pointing to the kubernetes docs makes more sense.

File: helm/idol-qms/values.schema.json (hunk 1 above)
Comment (christopher-rayson-mf) [reply helm/idol-qms/values.schema.json:92]: Oh, looks like I added these to the idol-view/values.yaml but missed QMS/omnigroupserver.

Comment (christopher-rayson-mf) [reply helm/idol-qms/values.schema.json:92]: 3784010

**Review by joe-wood-opentext — APPROVED** [review]

**Pull request merged**